apps/payments/next/app/[locale]/error.tsx

```diff
@@ -0,0 +1,120 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+'use client';
+
+import { useEffect, useState } from 'react';
+import * as Sentry from '@sentry/nextjs';
+import { useRouter, useParams } from 'next/navigation';
+import { CheckoutParams } from './[offeringId]/checkout/[interval]/[cartId]/layout';
+import Image from 'next/image';
+import errorIcon from '@fxa/shared/assets/images/error.svg';
+import { LoadingSpinner } from '@fxa/payments/ui';
+import Link from 'next/link';
+import { Localized } from '@fluent/react';
+import { restartCartAction, getCartAction } from '@fxa/payments/ui/actions';
+import { CartState } from '../../../../../libs/shared/db/mysql/account/src/lib/kysely-types';
```

Comment:
Could this be updated to use the TS alias?

Reply:
There is a similar issue to the `lib/actions/index.ts` barrel file. The other exported members of `db/mysql/account/src` make use of the `fs` package, which is server-only

Reply:
following off-github conversation: shift to direct string comparison, in an effort to avoid unaliased route imports